Casks/d/db-browser-for-sqlite.rb

```diff
@@ -1,18 +1,34 @@
 cask "db-browser-for-sqlite" do
-  version "3.13.0"
-  sha256 "dfa72811ab9faa522586a31bf680db1604442e35a2725f0aed77d5f66388724b"
+  arch arm: "-arm64"
+
+  on_catalina :or_older do
+    version "3.12.2"
+    sha256 arm:   "0c2076e4479cb9db5c85123cfe9750641f92566694ff9f6c99906321a2c424e8",
+           intel: "546d57b6c88c2be7517759c016c0bf0313dfcc14adfcb43967f3c5d24657f366"
+
+    url "https://github.com/sqlitebrowser/sqlitebrowser/releases/download/v#{version}/DB.Browser.for.SQLite#{arch}-#{version}.dmg"
```

Comment:
```suggestion
    url "https://github.com/sqlitebrowser/sqlitebrowser/releases/download/v#{version}/DB.Browser.for.SQLite#{arch}-#{version}.dmg",
        verified: "github.com/sqlitebrowser/sqlitebrowser/"
```

We also have to add the `verified` parameter to this `url` stanza.

Reply:
Cool, I've changed. :)